admin_guide/http_proxies.adoc

```diff
@@ -0,0 +1,117 @@
+= Working with HTTP Proxies
+{product-author}
+{product-version}
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:description: This should be a clear and concise description of the topic.
+:keywords: comma, separated, keywords, list
+
+toc::[]
+
+== Overview
+Production environments can deny direct access to the web and instead have an
+HTTP or HTTPS proxy available. Configuring OpenShift builds and deployments to
+use these proxies can be as simple as setting standard environment variables in
+configuration or JSON files.
+
+== Configuring Hosts for Proxies
+
+Add the `*NO_PROXY*`, `*HTTP_PROXY*`, and `*HTTPS_PROXY*` environment variables
+to the *_/etc/sysconfig/openshift-master_* and *_/etc/sysconfig/openshift-node_*
+files:
+
+====
+----
+HTTP_PROXY=http://USERNAME:PASSWORD@10.0.1.1:8080/
+HTTPS_PROXY=https://USERNAME:PASSWORD@10.0.0.1:8080/
+NO_PROXY=master.hostname.example.com
+----
+====
+
+== Proxying Docker Pull
+
+Docker pulls content from the registry that hosts your built images. If you have
+an internal registry then you must blacklist that registry's address from
+access via the external http proxy.
+
+. Edit the *_/etc/sysconfig/docker_* file and add the variables in shell format:
++
+====
+----
+HTTP_PROXY=http://USERNAME:PASSWORD@10.0.1.1:8080/
+HTTPS_PROXY=https://USERNAME:PASSWORD@10.0.0.1:8080/
+NO_PROXY=master.hostname.example.com
+----
+====
+
+. Restart the master service:
++
```

Comment:
@codificat That makes sense. I've added a step in the "hosts" section about restarting the appropriate host. 

Also, do you mean that in the "Docker" steps the master does not need to be restarted? I added that step (what it currently looks like now) after suggestion from @rimolive . Are you suggesting that they are not relevant to this section?